packages/Webkul/Velocity/src/Resources/views/shop/customers/account/orders/view.blade.php

```diff
@@ -28,11 +28,26 @@
 
                 @if ($order->canCancel())
                     <span class="account-action">
-                        <a href="{{ route('customer.orders.cancel', $order->id) }}" class="theme-btn light unset pull-right" v-alert:message="'{{ __('shop::app.customer.account.order.view.cancel-confirm-msg') }}'" style="float: right">
+                        <a href="{{ route('customer.orders.cancel', $order->id) }}" class="theme-btn light unset pull-right" v-alert:message="'{{ __('shop::app.customer.account.order.view.cancel-confirm-msg') }}'" style="float: right; margin-left: 8px;">
                             {{ __('shop::app.customer.account.order.view.cancel-btn-title') }}
                         </a>
                     </span>
                 @endif
+                @if (core()->getConfigData('customer.settings.reorder.product_reorder'))
+                    @if($can_reorder)
```

Comment:
Use space after if and Use camel case for $can_reorder

Reply:
done